lib/EscriptGrammar/EscriptParser.g4

```diff
@@ -377,6 +378,17 @@ literal
     | STRING_LITERAL
     ;
 
+interpolatedString
```

Comment:
The parser updates are primarily based off C# here: https://github.com/antlr/grammars-v4/blob/master/csharp/CSharpParser.g4 , minus the use of expression lists